.gitignore

```diff
@@ -1,2 +1,3 @@
+*.DS_Store
+xcuserdata
```

Comment:
This won't ignore the directory afaik. I think you have to add the full path for that directory.